lib/bolt/cli.rb

```diff
@@ -229,9 +229,9 @@ def update_targets(options)
 
     def validate(options)
       unless COMMANDS.include?(options[:subcommand])
+        command = Bolt::Util.powershell? ? 'Get-Command -Module PuppetBolt' : 'bolt help'
```

Comment:
This doesn't work if the user hasn't loaded the powershell module. I think we can see if the user has the module loaded with `Get-Module -ListAvailable -Name PuppetBolt` and switch on that instead?

Reply:
Shouldn't the powershell module be loaded automatically when Bolt is installed as a package?

Reply:
https://puppet.com/docs/bolt/latest/bolt_installing.html#add-the-bolt-module-to-powershell I doubt many people are using Powershell 2 or 3 but it generally just seems better to verify the cmdlets are loaded before recommending a cmdlet? Does `Get-Module` add a lot of time? There might be more efficient ways to check...

Reply:
Ok, it looks like in the last month, out of 278 Windows users we've had 7 on Windows 7 and Windows 2012 (non-r2), which have Powershell3 by default. I feel ok leaving this as is and modifying if anyone complains.